src/content/app/species/components/species-title-area/SpeciesTitleArea.scss

```diff
@@ -32,12 +32,12 @@ $container-padding-right: 20px;
 
 .speciesTitleArea {
   display: grid;
-  grid-template-areas:
-    'icon species-name usage-toggle remove';
+  grid-template-areas: 'icon species-name usage-toggle remove';
   grid-template-columns: 60px fit-content(60%) 200px minmax(280px, 1fr);
   grid-column-gap: 20px;
   align-items: center;
   height: 90px;
+  max-width: 1321px;
```

Comment:
This is a peculiarly precise max-width, which I don't think we've ever seen before. Why is it like this?

Reply:
Should we wrap these two in a container then, if we want to keep their right border the same?
 
![image](https://user-images.githubusercontent.com/6834224/182558495-82b72898-5c7a-4644-bd10-b2f78990df64.png)

Reply:
Agreed. i was trying to align both the titlebar and the stats box. I have now moved the max-width to the container.